avro/src/main/java/com/fasterxml/jackson/dataformat/avro/schema/RecordVisitor.java

```diff
@@ -190,6 +205,9 @@ public void optionalProperty(String name, JsonFormatVisitable handler,
             }
         }
         JsonNode defaultValue = parseJson(prop.getMetadata().getDefaultValue());
```

Comment:
Since you can no longer omit a default when this feature is on to get no default, we should probably add support for explicitly setting no default if `prop.getMetadata().getDefaultValue()` returns the empty string so that there is a way to specify "no default" even when this feature is enabled.

Reply:
Not if you want the field to be _nullable_, just not have a _null default value_. `@JsonProperty(required=true)` will make the field non-nullable.

Reply:
I see. Can usage of `@AvroSchema` might be sufficient for such use case?

Reply:
@baharclerode - Frankly I am not sure whether there is some action required from my side. Can you please ping me whether I should rewrite proposed change or my previous comment is sufficient?

Reply:
`@AvroSchema` should be sufficient-- This looks good to me!

Reply:
@cowtowncoder - Is there any activity from my side I should do?